### PR TITLE
Add dockerFile for the Docker image used in node discovery

### PR DESCRIPTION
**Pull request by keedya**
depends on : https://github.com/RackHD/on-http/pull/392
depends on : https://github.com/RackHD/on-taskgraph/pull/149
depends on : https://github.com/RackHD/on-tasks/pull/293

ref:  https://osclouds.wordpress.com/2015/11/23/announcing-a-new-docker-based-hanlon-microkernel
      https://groups.google.com/forum/#!topic/rackhd/cSYYxhtPWY0

### CODE REVIEW

**Comment by keedya** [comment]
@RackHD/corecommitters @zyoung51 @jlongever @benbp

**Comment by yyscamper** [comment]
:+1:

**Comment by benbp** [comment]
👍

File: micro-docker/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- cd micro-docker
+- sudo docker build -t rackhd/micro .
+- sudo docker save rackhd/micro | xz -z > micro.tar.xz
+- copy micro.tar.xz to on-http/static/http
```
Comment (benbp) [inline micro-docker/README.md]: This still needs to be changed to match the on-tasks name, right? https://github.com/RackHD/on-tasks/pull/293#issuecomment-236190792

**Pull request merged**